Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -44,6 +44,115 @@ static wstring __GetUserDefaultLocaleName() {
 }
 
 /**
+ * Private helper that creates an uppercase copy of a CFString
+ */
+CFStringRef __CFStringCreateUppercaseCopy(CFStringRef string, CFLocaleRef locale) {
+    CFMutableStringRef ret = CFStringCreateMutableCopy(nullptr, CFStringGetLength(string), string);
+    CFStringUppercase(ret, locale);
+    return ret;
+}
+
+/**
+ * Private helper that creates the IDWriteFontList for a given font family name
+ */
+HRESULT __DWriteGetFontListForFamilyName(CFStringRef familyName, IDWriteFontList** outFontList) {
+    ComPtr<IDWriteFactory> dwriteFactory;
+    RETURN_IF_FAILED(DWriteCreateFactory(DWRITE_FACTORY_TYPE_SHARED, __uuidof(IDWriteFactory), &dwriteFactory));
```

Comment:
I mentioned this on a previous PR - @bbowman stated that factory objects such as this, in the windows api world, often already implement their own reuse on the back end, and that by keeping one as a static, we would be interfering with their reusage patterns.

---
In reply to: [87900429](https://github.com/Microsoft/WinObjC/pull/1366#discussion_r87900429) [](ancestors = 87900429)